src/api.rs

```diff
@@ -1308,8 +1308,19 @@ fn handle_req<W: Write>(
         handle.progress(true)?;
     }
 
-    // enable verbose mode
-    handle.verbose(true)?;
+    if Config::current().get_log_level() == log::LevelFilter::Debug {
```

Comment:
There is also a `trace` log level.

```suggestion
    if Config::current().get_log_level() >= log::LevelFilter::Debug {
```